### DISCUSSION TITLE
表头斜线

### DISCUSSION THREAD

**Discussion by yemo0399** [note286/xduts#93] [Q&A] [answered]
求助！希望表头是有斜线分割，如下
![微信图片_20230323163854](https://user-images.githubusercontent.com/128452297/227148662-b8b6fc82-92be-4455-9f0a-d755b066d71e.png)

运行之后报错，没有斜线
![微信图片_20230323163910](https://user-images.githubusercontent.com/128452297/227148856-e06a3a1e-c974-49db-836f-c7654e10dd69.png)

![微信图片_20230323163936](https://user-images.githubusercontent.com/128452297/227148797-5a62201e-4804-4e4f-9a9f-fc35becea2ea.png)

**Comment by sikouhjw** [comment]
宏包加载了没？报什么错？

**Reply by yemo0399** [reply]
加载上了，半天才注意到！谢谢谢谢！！！

**Comment by note286** [comment]
看`diagbox`文档，包加载了吗？

**Reply by yemo0399** [reply]
加载上了，查了半天没注意到！谢谢谢谢！！！

**Comment by note286** [comment, marked as answer]
另外，根据 CYT 170-2019 学术出版规范 表格：

![image](https://user-images.githubusercontent.com/16207292/227149712-c65b229f-c864-4bd8-aa1f-a4e3c2e55f88.png)

**Reply by yemo0399** [reply]
好的好的，谢谢大佬，不知道这个规范
